### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 13th of June 2023 17:30 CEST / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#1068] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 13th of June 2023 17:30 CEST / 11:30am EDT / 8:30am PDT

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: [here](https://www.youtube.com/watch?v=lxbH2zyQLWI)
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items
- @pothos to demo Azure OEM sysext
- @t-lo KCD Munich 

### Release Planning
- https://github.com/orgs/flatcar/projects/7/views/8

### Administrative tasks
- Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- Close this discussion as "Resolved".

**Comment by miao0miao** [comment]
Activity on the last 30 days - issues on Flatcar repo:

## Issues Opened
* [#1069](https://github.com/flatcar/Flatcar/issues/1069):  update: go
* [#1065](https://github.com/flatcar/Flatcar/issues/1065):  mantle: DigitalOcean image removal
* [#1063](https://github.com/flatcar/Flatcar/issues/1063):  If I want to compile something to run on flatcar, what is the most appropriate linux environment to do so?
* [#1062](https://github.com/flatcar/Flatcar/issues/1062):  [RFE] sshd default config should read drop-ins
* [#1061](https://github.com/flatcar/Flatcar/issues/1061):  CAPI images through systemd-sysext (system extensions) for easy CAPI updates
* [#1060](https://github.com/flatcar/Flatcar/issues/1060):  [RFE] Harden Flatcar Linux Defaults Further
* [#1059](https://github.com/flatcar/Flatcar/issues/1059):  Install script not working correctly on the latest versions
* [#1055](https://github.com/flatcar/Flatcar/issues/1055):  Unable to allow root privileges for specific group and block all others
* [#1054](https://github.com/flatcar/Flatcar/issues/1054):  update: libarchive
* [#1053](https://github.com/flatcar/Flatcar/issues/1053):  update: binutils
* [#1052](https://github.com/flatcar/Flatcar/issues/1052):  Propose a mechanism to enable sysext builds in the OS image build process
* [#1051](https://github.com/flatcar/Flatcar/issues/1051):  Booting issue of Alpha 3619.0.0 for Equinix Metal arm64
* [#1050](https://github.com/flatcar/Flatcar/issues/1050):  update: openssl
* [#1049](https://github.com/flatcar/Flatcar/issues/1049):  [RFE] upgrade coreos-base/afterburn
* [#1048](https://github.com/flatcar/Flatcar/issues/1048):  Reboot hang on hw watchdog handling
* [#1047](https://github.com/flatcar/Flatcar/issues/1047):  scripts build and test: sdk builds via GitHub workflow
* [#1046](https://github.com/flatcar/Flatcar/issues/1046):  scripts build and test: fix report generation, parallelisation
* [#1045](https://github.com/flatcar/Flatcar/issues/1045):  update: ncurses
* [#1044](https://github.com/flatcar/Flatcar/issues/1044):  amd64: arm64: hang during file descriptor close
* [#1041](https://github.com/flatcar/Flatcar/issues/1041):  Release Flatcar Container Linux Alpha 3619.0.0, Beta 3602.1.0, Stable 3510.2.2, LTS 3033.3.13
* [#1040](https://github.com/flatcar/Flatcar/issues/1040):  Proxmox support
* [#1038](https://github.com/flatcar/Flatcar/issues/1038):  update: sudo
* [#1036](https://github.com/flatcar/Flatcar/issues/1036):  Extend Flatcar quickstart guide
* [#1035](https://github.com/flatcar/Flatcar/issues/1035):  [RFE] Update the default Go to 1.20
* [#1034](https://github.com/flatcar/Flatcar/issues/1034):  NVIDIA GPUs: add documentation on how to use (for users), and pointers to ebuilds (for devs)
* [#1032](https://github.com/flatcar/Flatcar/issues/1032):  Extension image signing for update-engine
* [#1031](https://github.com/flatcar/Flatcar/issues/1031):  Clock skew after upgrading from 3374.2.3 to 3510.2.1
* [#1030](https://github.com/flatcar/Flatcar/issues/1030):  Flatcar arm64 running on QEMU (kvm enabled) does not have video output when using VNC
* [#1029](https://github.com/flatcar/Flatcar/issues/1029):  [RFE] Add ignition support in kops
* [#1028](https://github.com/flatcar/Flatcar/issues/1028):  Integration of ue-rs with update-engine post-install script
* [#1027](https://github.com/flatcar/Flatcar/issues/1027):  Signature Verification
* [#1026](https://github.com/flatcar/Flatcar/issues/1026):  rustfmt configuration
* [#1025](https://github.com/flatcar/Flatcar/issues/1025):  Parse/decode CrAU protobuf format
* [#1023](https://github.com/flatcar/Flatcar/issues/1023):  99-default.rules reappear after update
* [#1020](https://github.com/flatcar/Flatcar/issues/1020):  Release Flatcar Container Linux Alpha 3602.0.0 Beta 3572.1.0

## Issues Closed
* [#1046](https://github.com/flatcar/Flatcar/issues/1046):  scripts build and test: fix report generation, parallelisation
* [#1038](https://github.com/flatcar/Flatcar/issues/1038):  update: sudo
* [#1035](https://github.com/flatcar/Flatcar/issues/1035):  [RFE] Update the default Go to 1.20
* [#1031](https://github.com/flatcar/Flatcar/issues/1031):  Clock skew after upgrading from 3374.2.3 to 3510.2.1
* [#1030](https://github.com/flatcar/Flatcar/issues/1030):  Flatcar arm64 running on QEMU (kvm enabled) does not have video output when using VNC
* [#1026](https://github.com/flatcar/Flatcar/issues/1026):  rustfmt configuration
* [#1023](https://github.com/flatcar/Flatcar/issues/1023):  99-default.rules reappear after update
* [#1020](https://github.com/flatcar/Flatcar/issues/1020):  Release Flatcar Container Linux Alpha 3602.0.0 Beta 3572.1.0
* [#1013](https://github.com/flatcar/Flatcar/issues/1013):  Where to find latest manifests?
* [#1008](https://github.com/flatcar/Flatcar/issues/1008):  Release Flatcar Container Linux Alpha 3572.0.1, Beta 3549.1.1, Stable 3510.2.1, LTS 3033.3.12
* [#1006](https://github.com/flatcar/Flatcar/issues/1006):  nvidia-drivers setup file missing from the source
* [#998](https://github.com/flatcar/Flatcar/issues/998):  flatcar-install script not work in alpine 3.17
* [#996](https://github.com/flatcar/Flatcar/issues/996):  Release Flatcar Container Linux Alpha 3572.0.0, Beta 3549.1.0, Stable 3510.2.0, LTS 3033.3.11
* [#987](https://github.com/flatcar/Flatcar/issues/987):  Support UpheldBy as systemd property for enabling units
* [#981](https://github.com/flatcar/Flatcar/issues/981):  locksmithd: update deprecated unit parameters
* [#968](https://github.com/flatcar/Flatcar/issues/968):  Butane vs ct
* [#828](https://github.com/flatcar/Flatcar/issues/828):  Ignition v2 support is broken in Flatcar 3227.2.0 or newer
* [#821](https://github.com/flatcar/Flatcar/issues/821):  Bonding issues in 3227.2.0 / systemd 250: Cannot enslave bond to itself
* [#790](https://github.com/flatcar/Flatcar/issues/790):  [RFE] Trigger the scripts CI GitHub Action workflow for coreos-overlay/portage-stable PRs
* [#777](https://github.com/flatcar/Flatcar/issues/777):  weekly testing failed on AWS
* [#769](https://github.com/flatcar/Flatcar/issues/769):  Implement the Azure OEM tools as systemd-sysext image
* [#766](https://github.com/flatcar/Flatcar/issues/766):  Move OEM mount point out of /usr
* [#756](https://github.com/flatcar/Flatcar/issues/756):  CT - custom update group name
* [#748](https://github.com/flatcar/Flatcar/issues/748):  Ignition: networkd unit not applied (Link aggregation config not applied as a clc snippet)
* [#559](https://github.com/flatcar/Flatcar/issues/559):  [RFE] Add variants for some kola tests that start from an empty rootfs
* [#480](https://github.com/flatcar/Flatcar/issues/480):  fstrim.service as shipped creates dependency on /etc/fstab which doesn't exist
* [#453](https://github.com/flatcar/Flatcar/issues/453):  Support all update.conf configuration in the config transpiler
* [#442](https://github.com/flatcar/Flatcar/issues/442):  (GitHub CI) Productise containerised build and release pipeline
* [#428](https://github.com/flatcar/Flatcar/issues/428):  migrate to 17.1 profiles
* [#427](https://github.com/flatcar/Flatcar/issues/427):  Kernel panics with use_after_free on 2765.2.4
* [#398](https://github.com/flatcar/Flatcar/issues/398):  [RFE] Upgrade Config Transpiler so it would support new ETCD versions
* [#65](https://github.com/flatcar/Flatcar/issues/65):  users cannot write system-shutdown scripts

**Comment by tormath1** [comment]
A few members of the team will be present at KCD Munich - we can briefly mention the Flatcar related talks and call for meeting folks there. :)
